cpp/src/arrow/compute/kernels/scalar_temporal_unary.cc

```diff
@@ -1801,8 +1831,9 @@ const FunctionDoc ceil_temporal_doc{
 const FunctionDoc round_temporal_doc{
     "Round temporal values to the nearest multiple of specified time unit",
     ("Null values emit null.\n"
-     "An error is returned if the values have a defined timezone but it\n"
-     "cannot be found in the timezone database."),
+     "If timezone is not given then timezone naive timestamp in UTC are\n"
+     "returned. An error is returned if the values have a defined timezone\n"
+     "but it cannot be found in the timezone database."),
```

Comment:
Was this change intentional? Or was it meant to be in `local_time_doc`? 
(and if intentional, why only for round and not ceil/floor?)